### DISCUSSION TITLE
Thank you for the awesome service

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1293] [Kind words ❤️]
Thank you so much for the amazing service! I am just thrilled.

SimpleLogin has saved me a lot of nerves when I need to communicate with various suspicious individuals, intrusive customers and spammers. Thanks to the service, I was able to simply delete the problematic alias and that was the end of all problems.

But I have one question:
What happens to a SimpleLogin account if you don't use it for a long time? Don't you have an inactivity policy?

Thanks again!

**Comment by nguyenkims** [comment]
Thanks for your kind words :). Currently nothing happens to your account if you don't use it. If our policy ever changes in the future, we'll make sure to announce this change on our website and via email.
